sass/directives/02_base_components/button/_button.scss

```diff
@@ -222,6 +222,101 @@ $massive-button-horizontal-padding: $base-input-font-size * 2;
   }
 }
 
+// Fancy Play Button
+// Initial state
+$ripple-diameter: 24em;
+$play-button-size: 4em;
+$ripple-color: white;
+$ripple-hover-color: #447777;
+$ripple-hover-transition: 1500ms ease;
```

Comment:
take a look at the [animation documentation](http://cb-talent-development.github.io/employer-style-base/components/detail/animations--transition-base.html) and consider how this should be documented ther

* should `$ripple-hover-transition`, `$ripple-duration`, and `$ripple-count` go in the variables/animations style sheet?
* should generic variables be defined in animations.scss that are then used in place of the number values here?